types/CallToAction.ts

```diff
@@ -0,0 +1,8 @@
+type CallToAction = {
```

Comment:
This type is the same as the type you've defined in CallToActionButton

I'm assuming this is generated by sanity, but let's reuse it! That saves you from having to worry about type matching.

Reply:
I'm now importing this type into the `CallToAction` component so there's no repetition.